source/frontend/src/components/common/SectionListHeader.tsx

```diff
@@ -12,6 +12,7 @@ export interface ISectionListHeaderProps {
   addButtonIcon?: JSX.Element;
   onAdd?: () => void;
   claims: Claims[];
+  dataTestid?: string;
```

Comment:
fyi: this is the way to make this consistent with the standard naming of this prop: 'data-testId': string;